src/main/java/org/mifos/pheevouchermanagementsystem/api/definition/CancelAndRedeemVoucherApi.java

```diff
@@ -0,0 +1,21 @@
+package org.mifos.pheevouchermanagementsystem.api.definition;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestHeader;
+import org.springframework.web.bind.annotation.RequestParam;
+
+@Tag(name = "GOV")
+public interface CancelAndRedeemVoucherApi {
+
+    @Operation(summary = "Cancel or Redeem Voucher API")
+    @PostMapping("/vouchers")
+    <T> ResponseEntity<T> cancelOrRedeemVoucher(@RequestHeader(value = "X-CallbackURL") String callbackURL,
```

Comment:
link the PR for documentation(POSTMAN) updation with this API

Reply:
Sure.